bin/init.bash

```diff
@@ -235,18 +238,18 @@ set_nginx_config() {
             ;;
     esac
 
-    replace_set_me "$1" '.ingressNginx.controller.config.useProxyProtocol' "${use_proxy_protocol}"
-    replace_set_me "$1" '.ingressNginx.controller.useHostPort' "${use_host_port}"
-    replace_set_me "$1" '.ingressNginx.controller.service.enabled' "${service_enabled}"
-    replace_set_me "$1" '.networkPolicies.global.externalLoadBalancer' "${external_load_balancer}"
-    replace_set_me "$1" '.networkPolicies.global.ingressUsingHostNetwork' "${ingress_using_host_network}"
+    replace_set_me "$file" '.ingressNginx.controller.config.useProxyProtocol' "${use_proxy_protocol}"
```

Comment:
Nit
```suggestion
    replace_set_me "${file}" '.ingressNginx.controller.config.useProxyProtocol' "${use_proxy_protocol}"
```

Reply:
@aarnq please have a look at the [latest commit](https://github.com/elastisys/compliantkubernetes-apps/pull/1845/commits/5e5bc96d618870595343c895603b9be0055f1afe), resolve this if it looks fine and I can close this PR

Reply:
Looks good! I even got more than I asked for :smile: